content/02.main-text.md

```diff
@@ -14,10 +14,8 @@ However, inviting wide authorship brings many technical and social challenges su
 
 We present solutions to these challenges based on our recent experience leading a collaborative review "Opportunities And Obstacles For Deep Learning In Biology And Medicine" [@doi:10.1101/142760].
 Our review attracted 27 authors from 20 different institutions who were not determined in advance.
-`TODO: confirm institution count`
 We wrote entirely in the open without restrictions on who was welcome to contribute.
-Although we requested that some authors participate for their specific expertise, most discovered the manuscript organically through conferences or social media and independently decided to contribute.
-`TODO: confirm "most"`
+Although we requested that some authors participate for their specific expertise, most seem to have discovered the manuscript organically through conferences or social media and independently decided to contribute.
```

Comment:
In the name of persuasive writing, I think we can strip "seem to have"

Reply:
Agree we can remove "seem to have".  @cgreene and I discussed this after his commit.  Our author counts are fuzzy but do support "most".

Reply:
👍